test/merchant_repository_test.rb

```diff
@@ -0,0 +1,25 @@
+require './test/test_helper'
+require 'csv'
+require './lib/merchant_repository'
+require 'pry'
```

Comment:
To clean up the requires, could you remove the pry reference?

Reply:
Good catch! Removed pry